app/styles/modules/_custom-rows.scss

```diff
@@ -86,6 +86,16 @@ ul.links {
 }
 
 .choose-what-to-sync-row {
+  &:active{
+    .fxa-checkbox__box-outline{
+      border: 2px solid $checkbox-border-color;
+
+      .fxa-checkbox__tick-outline{
+        background: url("/images/tick.svg");
```

Comment:
@vbudhram can we include these changes in fxa-checkbox instead?  feel free to remove the encoded svg in it and replace it with a new one